client/client_proxy.go

```diff
@@ -53,24 +54,39 @@ func (c *MockLogClient) GetInclusionProof(ctx context.Context, in *trillian.GetI
 // GetInclusionProofByHash forwards requests.
```

Comment:
(pre-existing) Comment is inaccurate -- the method does more than just forwarding requests